cmdstanpy/cmdstan_args.py

```diff
@@ -132,7 +132,7 @@ def validate(self, chains: int) -> None:
                 )
         if self.step_size is not None:
             if isinstance(self.step_size, Real):
-                if self.step_size < 0:
+                if self.step_size <= 0:
```

Comment:
must be strictly greater than zero - https://github.com/stan-dev/cmdstan/blob/3e93d44ca33200bb3f09218451ffa36ab3c8a01d/src/cmdstan/arguments/arg_stepsize.hpp#L13

Reply:
I think it was wrong

If `init_alpha` was 0, it would have passed, but now it is catched.

See

    0 < init_alpha

vs 

    init_alpha <= 0

Reply:
doh!  you are both correct - got the comparison backward.